docs/wiki/PipelinesDesign.md

```diff
@@ -6,34 +6,44 @@ This section gives you an overview of the design principals the pipelines follow
 
 ### _Navigation_
 
+- [Pipelines Design](#pipelines-design)
+    - [_Navigation_](#navigation)
 - [Module Pipelines](#module-pipelines)
   - [Module pipeline inputs](#module-pipeline-inputs)
   - [Pipeline phases](#pipeline-phases)
     - [Validate](#validate)
+      - [Static module validation](#static-module-validation)
+      - [Simulated deployment validation](#simulated-deployment-validation)
     - [Test deploy](#test-deploy)
       - [Removal](#removal)
     - [Publish](#publish)
   - [Shared concepts](#shared-concepts)
     - [Prerequisites](#prerequisites)
     - [Pipeline secrets](#pipeline-secrets)
     - [Pipeline variables](#pipeline-variables)
+      - [**_General_**](#general)
+      - [**_Template-specs specific (publishing)_**](#template-specs-specific-publishing)
+      - [**_Private bicep registry specific (publishing)_**](#private-bicep-registry-specific-publishing)
     - [Tokens Replacement](#tokens-replacement)
 - [Platform pipelines](#platform-pipelines)
   - [Dependencies pipeline](#dependencies-pipeline)
+    - [Dependencies pipeline inputs](#dependencies-pipeline-inputs)
   - [ReadMe pipeline](#readme-pipeline)
   - [Wiki pipeline](#wiki-pipeline)
 - [DevOps-Tool-specific considerations](#devops-tool-specific-considerations)
   - [GitHub Workflows](#github-workflows)
-    - [Component: GitHub secret](#github-component-github-secrets)
-    - [Component: Variable file](#github-component-variable-file)
-    - [Component: Composite actions](#github-component-composite-actions)
-    - [Component: Workflows](#github-component-workflows)
+    - [**GitHub Component:** GitHub secrets](#github-component-github-secrets)
+    - [**GitHub Component:** Variable file](#github-component-variable-file)
+    - [**GitHub Component:** Composite Actions\*\*](#github-component-composite-actions)
+    - [**GitHub Component:** Workflows](#github-component-workflows)
   - [Azure DevOps Pipelines](#azure-devops-pipelines)
-    - [Component: Service connection](#azure-devops-component-service-connection)
-    - [Component: Variable group](#azure-devops-component-variable-group)
-    - [Component: Variable file](#azure-devops-component-variable-file)
-    - [Component: Pipeline templates](#azure-devops-component-pipeline-templates)
-    - [Component: Pipelines](#azure-devops-component-pipelines)
+    - [**Azure DevOps Component:** Service Connection](#azure-devops-component-service-connection)
```

Comment:
```suggestion
    - [**Component:** Service Connection](#azure-devops-component-service-connection)
```
why all of those longer when they are already summarized under one common header?

Reply:
Reverted whole TOC